### PR TITLE
Lactic ferments

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -6162,7 +6162,7 @@ vegetarian:en:yes
 # comment:en:Should this be kept separate from lactic ferments?
 
 <en:ferment
-en:microbial culture, culture
+en:microbial culture, culture, activ cultures, live cultures
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
en:microbial culture, culture, active cultures, live cultures
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -32734,7 +32735,8 @@ cs:Borůvka
 da:blåbær
 de:Heidelbeer, Heidelbeeren, Heidelbeere
 eo:mirtelo
-es:arándano azul
+es:arándano
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: I think we should put all of them as synonyms:


```suggestion
es:arándano, arándano azul, mora azul
```

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -32734,7 +32735,8 @@ cs:Borůvka
...
+es-mx:mora azul
```
Comment (VaiTon) [inline taxonomies/ingredients.txt]:
```suggestion
```

**Comment by moon-rabbitOFF** [comment]
I tried to apply your changes but I didn't succeded

**Pull request merged**